config/jobs/kubernetes/sig-node/node-kubelet.yaml

```
@@ -456,7 +456,7 @@ periodics:
           - --node-test-args=--kubelet-flags="--exit-on-lock-contention --lock-file=/var/run/kubelet.lock"
           - --node-tests=true
           - --provider=gce
-          - --test_args=--nodes=1 --focus="\[NodeFeature:LockContention\]" --skip=""
+          - --test_args=--nodes=1 --focus="\[NodeSpecialFeature:LockContention\]" --skip="\[Flaky\]|\[Serial\]"
```

```suggestion
          - --test_args=--nodes=1 --focus="\[NodeSpecialFeature:LockContention\]" --skip="\[Flaky\]"
```

If these tests need to be run serially, we should include that on them.

---

This test need not be run serially, as the test is disruptive and restarts Kubelet.